stock_picking_analytic/models/stock_picking.py

```diff
@@ -37,18 +34,40 @@ def _compute_analytic_distribution(self):
         """
         Get analytic account from first move and put it on picking
         """
+        if not self.filtered("id"):
+            return
+        self.flush_model(["move_ids", "original_analytic_distribution"])
+        self.move_ids.flush_model(["analytic_distribution"])
```

Comment:
This will trigger a read on the field.

Reply:
:laughing: it's still reading the field.

Just after you access `self.move_ids`, it's reading the field.

Reply:
Third time's a charm